x-pack/plugins/enterprise_search/public/applications/shared/schema/add_field_modal/constants.tsx

```diff
@@ -10,6 +10,8 @@ import React from 'react';
 import { i18n } from '@kbn/i18n';
 import { FormattedMessage } from '@kbn/i18n/react';
 
+export const FORM_ID = 'schemaAddFieldForm';
```

Comment:
Just curious, why did you decide to move this to a shared constants file rather?

Reply:
No super strong reason, it's only used in 2 places but I figured I'd pull it out because

1. I can't see a scenario where someone would intentionally update the ID of one usage but not the other, meaning if it did get updated in the future it would be easier/more helpful to only update 1 reference
2. Pulling it out also highlights the importance of the form `id` / how the modal footer button submits a totally separate form